src/ru/geekbrains/lesson6/Animal.java

```diff
@@ -0,0 +1,26 @@
+package ru.geekbrains.lesson6;
+
+abstract public class Animal {
+    protected static int animalsCounter=0;
+    public abstract void makeSound();
+
+    public static void showAnimalCount(){
+        System.out.println("всего животных: "+ animalsCounter);
+    }
+
+    public String getClassName() {
+        return this.getClass().getSimpleName();
+    }
+
+    interface CanRun {
+        void run(int distance);
+    }
+    interface CanSwim{
+        void swim(int distance);
+    }
+    interface CanJump{
```

Comment:
```suggestion
    interface Jumpable {
```
глаголы лучше не использовать в именах классов/интерфейсов